api/imagescan.go

```diff
@@ -18,6 +18,7 @@ import (
 	"net/http"
 	"path"
 
+	"github.com/banzaicloud/pipeline/api/ark/common"
```

Comment:
Is there any specific reason that we import this from api/ark/common and not refactor it to api/common?

Reply:
Agree this should go into common.

Reply:
I'm on it